### PR TITLE
remove DEBUG macro in favor of LOG

### PR DESCRIPTION
**Pull request by MichaelBrim**
Now that we have `LOG()` available to client code, there's no need for `DEBUG()`.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [x] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation (a change to man pages or other documentation)

### CODE REVIEW

File: client/src/unifycr-fixed.c [hunk 1]
```diff
@@ -363,6 +364,7 @@ static int unifycr_logio_chunk_write(
         ssize_t rc = __real_pwrite(unifycr_spilloverblock, buf, count, spill_offset);
         if (rc < 0)  {
             perror("pwrite failed");
+            LOGERR("pwrite failed");
```
Comment (adammoody) [inline client/src/unifycr-fixed.c]: Let's also add the errno/strerror(errno) info that we'd get from perror(), e.g,.:

LOGERR("pwrite failed: errno=%d %s", errno, strerror(errno))

File: client/src/unifycr-fixed.c [hunk 2]
```diff
@@ -492,12 +494,13 @@ static int unifycr_chunk_write(
         ssize_t rc = pwrite(unifycr_spilloverblock, buf, count, spill_offset);
         if (rc < 0)  {
             perror("pwrite failed");
+            LOGERR("pwrite failed");
```
Comment (adammoody) [inline client/src/unifycr-fixed.c]: and here too

File: client/src/unifycr-fixed.c (hunk 1 above)
Comment (adammoody) [reply client/src/unifycr-fixed.c]: That's a good idea.  I'm good with that.

Comment (MichaelBrim) [reply client/src/unifycr-fixed.c]: Latest pushed version has these changes. Ready for review/merge.

**Comment by adammoody** [comment]
Thanks, @MichaelBrim !

**Pull request merged**